EasyPost/CreditCard.cs

```diff
@@ -81,7 +81,7 @@ private static async Task<CreditCardFund> Fund(string amount, string paymentMeth
                 }
             });
 
-            return await request.Execute<CreditCardFund>();
+            return await request.Execute();
```

Comment:
@nwithan8 wouldn't we want to return the bool only after executing instead of returning the request itself.

Reply:
`Request.Execute()` (without a class type) doesn't attempt to serialize the JSON response into any object. Instead, it returns a boolean based on whether an error was thrown during the HTTP request:
<img width="715" alt="image" src="https://user-images.githubusercontent.com/17054780/176510654-dfd740ab-cdb1-4fd7-a1d1-9923531cb864.png">

So, the result of `await request.Execute()` will be `true` if the API call was successful, `false` if it wasn't (a 4XX or 5XX was thrown)

Reply:
This was added a while ago because a lot of our set-and-forget API calls (mostly DELETE calls) were hard-coded to return `true`, even if an HTTP error occurred during the API call.
Now, the return value is based on whether the API call was successful.

Reply:
I remember now, thanks!